test/e2e/config/ironic.yaml

```diff
@@ -33,7 +33,7 @@ intervals:
   default/wait-registering: ["1m", "5s"]
   inspection/wait-registration-error: ["1m", "5s"]
   external-inspection/wait-available: ["20s", "1s"]
-  default/wait-inspecting: ["2m", "10s"]
+  default/wait-inspecting: ["2m", "2s"]
```

Comment:
any reason to change this in this PR ?

Reply:
Using 10-second intervals means that it can miss the state, so I changed it to 2 to one make the test more robust.

Reply:
You are right to ask because I now see that this is in the ironic config, but after trying it out I noticed some flakes here so that why this fix is in this PR even that it has nothing to do with fixture.